powerauth-webflow-authentication-mtoken/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/mtoken/controller/MobileAppApiController.java

```diff
@@ -313,9 +318,10 @@ private boolean verifyActivationId(String activationId, String userId) {
     private boolean isSignatureTypeAllowedForOperation(String operationName, PowerAuthSignatureTypes signatureTypes)  {
 
         // Get configuration for operation with given name
-        final GetOperationConfigResponse operationConfig = getOperationConfig(operationName);
-
-        if (operationConfig == null) {
+        GetOperationConfigResponse operationConfig;
+        try {
+            operationConfig = getOperationConfig(operationName);
+        } catch (AuthStepException e) {
```

Comment:
Should we add logging here?

Reply:
This is intentional, the Exception would be logged twice.

Reply:
ok